tests/Resources/Page/OCPDashboard/Pods/Pods.robot

```diff
@@ -51,3 +51,38 @@ Get Pod Status
     ${data}       Run Keyword   OpenShiftCLI.Get   kind=Pod
     ...    namespace=${namespace}   label_selector=${label_selector}
     [Return]      ${data[0]['status']['phase']}
+
+Get POD Names
+    [Documentation]    Get the name of list based on
+    ...    namespace and label selector and return the
+    ...    name of all the pod with matching label selector
+    [Arguments]   ${namespace}   ${label_selector}
+    ${pod_name}    Create List
+    ${status}      Check If POD Exists       ${namespace}        ${label_selector}
+    IF    '${status}'=='PASS'
+         ${data}        OpenShiftCLI.Get   kind=Pod     namespace=${namespace}   label_selector=${label_selector}
+         FOR    ${index}    ${element}    IN ENUMERATE    @{data}
+                Append To List    ${pod_name}     ${data[${index}]['metadata']['name']}
+         END
+    ELSE
+         FAIL    No POD found with the provided label selector in a given namespace '${namespace}'
+    END
+    [Return]    ${pod_name}
+
+Get Container Restart Counts
+    [Documentation]    Get the container name with restart
+    ...    count for each pod provided
+    [Arguments]        ${name}   ${namespace}
+    ${restart_c}      Create Dictionary
+    FOR    ${element}    IN    @{name}
```

Comment:
I would change `element` and `name` variable names. They are not very descriptive. `name` should be a list of pod names right?